addons/road-generator/plugin.gd

```diff
@@ -1257,6 +1272,13 @@ func _add_next_rp_on_click_do(pos: Vector3, nrm: Vector3, selection: Node, paren
 				look_pos += selection.global_transform.basis.z * selection.prior_mag
 			next_rp.look_at(look_pos, nrm)
 
+			if _lock_x_rotation:
+				next_rp.global_rotation.x = 0.0
+			if _lock_y_rotation:
+				next_rp.global_rotation.y = 0.0
+			if _lock_z_rotation:
+				next_rp.global_rotation.z = 0.0
```

Comment:
It seems like this it the main spot where locking the rotation like this would be useful, but maybe there are more places that I missed